src/components/EventList.jsx

```diff
@@ -4,16 +4,12 @@ import Moment from "moment";
 import "./styles/EventList.css";
 
 const Event = props => (
-  <div>
-    <p className="mb-0">{props.event.dateLabel}</p>
+  <div className="event-body">
     <p>
-      <span className="accent">
-        {props.event.title}
-        {!props.event.description || props.event.description.length === 0
-          ? ""
-          : ": "}
-      </span>
-      {props.event.description}
+      <strong>{props.event.dateLabel}</strong>
+      <br />
+      <span className="accent">{props.event.title}</span>
+      <div dangerouslySetInnerHTML={{__html: props.event.description}} />
```

Comment:
Unsanitized html, but it's taken from the Google Calendar API so I think we can trust it. Let me know there are any alternatives.

Reply:
![image](https://user-images.githubusercontent.com/22324472/81326800-5416cb80-9068-11ea-9bb5-cce1398430b2.png)
looks like it shows any html i try to add to a calendar event as text instead of parsed as actual HTML. Just wanted to make sure someone can't add something weird via calendar and it looks like it'll just display it plain text. So it does appear to only be google calendar's html that is unsanitized

(this is what a similar description looks like on master, google strips it to plain text)
![image](https://user-images.githubusercontent.com/22324472/81327140-d0a9aa00-9068-11ea-84c9-9ef8bbae5c83.png)

Reply:
yeah it's probably fine, I do this occasionally too